cli/lightning-cli.c

```diff
@@ -78,8 +78,10 @@ static size_t human_readable(const char *buffer, const jsmntok_t *t, char term)
 			return human_readable(buffer, t + 2, '\n') + 3;
 		n = 1;
 		for (i = 0; i < t->size; i++) {
-			n += human_readable(buffer, t + n, '=');
+			n += human_readable(buffer, t + n, '\t');
```

Comment:
A nice change, but it was done this way so 'lightning-cli -H listconfigs' gave a valid config.

While you're here giving the cli some much-needed love, can you make listconfigs output usable by default?  Thanks!

Reply:
I didn't realize it was used for this. I think I'll just revert this for now instead of trying to factor out two different human/config renderers.